### PR TITLE
Fix terminal on windows

### PR DESCRIPTION
**Pull request by abhiroopc84**
### Description
Fix the powershell terminal on Windows by adding arguments to allow Powershell to execute scripts without any restriction.

image

### What is the purpose of this pull request? 

- [ ] New feature
- [ ] Documentation update
- [X] Bug fix
- [ ] Refactor
- [ ] Release
- [ ] Other

### CODE REVIEW

File: apps/studio/src/routes/editor/EditPanel/WindowSettings/index.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import { Button } from '@onlook/ui/button';
```
Comment (Kitenite) [inline apps/studio/src/routes/editor/EditPanel/WindowSettings/index.tsx:1]: This may be a different PR's work? I think you're just intending to pass the Bypass?

Comment (abhiroopc84) [reply apps/studio/src/routes/editor/EditPanel/WindowSettings/index.tsx:1]: Yes, my bad
